mesheryctl/internal/cli/root/model/import.go

```diff
@@ -208,9 +201,271 @@ func sendToAPI(data []byte, name string, dataType string) error {
 		err = models.ErrDataRead(err, "response body")
 		return err
 	}
+	var response models.RegistryAPIResponse
 
-	bodyString := string(bodyBytes)
-	utils.Log.Info(bodyString)
-	utils.Log.Info("Models imported successfully")
+	if err := meshkitutils.Unmarshal(string(bodyBytes), &response); err != nil {
+		err = models.ErrUnmarshal(err, "response body")
+		return err
+	}
+	displayEntities(&response)
 	return nil
 }
+
+func displayEntities(response *models.RegistryAPIResponse) {
+	ok := displayEmtpyModel(response)
+	if !ok {
+		return
+	}
+	displaySummary(response)
+	displayEntitisIfModel(response)
+}
+func displayEmtpyModel(response *models.RegistryAPIResponse) bool {
+	if len(response.ModelName) != 0 && response.EntityCount.CompCount == 0 && response.EntityCount.RelCount == 0 {
+		if response.EntityCount.TotalErrCount == 0 {
+			return false
+		}
+	}
+	return true
+}
+
+// TO check the case if we were never able to read the file at first palce
+func hasExtension(name string) bool {
```

Comment:
Let's avoid using file extensions. They are fragile. We can sequentially try to parse them and figure out if they are of any valid format.

Reply:
Those are the scenarios where the file is corrupted and then we check for the extension. It is when the whole file had an issue and in that case the data returned is the filename as model so yes that is why we are doing this.